config/routes.rb

```diff
@@ -36,8 +36,18 @@
     get '/ping', to: 'ping#ping'
   end
 
+  namespace :support_interface, path: '/support' do
+    get '/' => redirect('/support/applications')
+
+    get '/applications' => 'applications#index'
+    get '/applications/:application_form_id' => 'applications#show', as: :application
```

Comment:
The word `application` is getting pretty overloaded: it's a `_form`, it's a `_choice` and sometimes (as in the presenter above) it's both. It feels like we might have the model a bit askew. Can we revisit?